src/js/spatial-navigation.js

```diff
@@ -56,12 +56,17 @@ class SpatialNavigation extends EventTarget {
     this.player_.on('modalclose', () => {
       this.refocusComponent();
     });
-    this.player_.on('error', () => {
-      this.focus(this.updateFocusableComponents()[0]);
-    });
     this.player_.on('focusin', this.handlePlayerFocus_.bind(this));
     this.player_.on('focusout', this.handlePlayerBlur_.bind(this));
     this.isListening_ = true;
+    this.player_.errorDisplay.on('aftermodalfill', () => {
+      this.updateFocusableComponents();
+
+      // Focus the buttons of the modal
+      if (this.focusableComponents.length > 1) {
+        this.focusableComponents[1].focus();
```

Comment:
- Why do we focus 2nd element here, not first?
- What if a modal has exactly one focusable element? (this if statement does not cover this case)